### ISSUE TITLE
All the files in the extension tutorial pages are returning 4-4

### ISSUE THREAD

**Issue by sarsntony** [GoogleChrome/developer.chrome.com#503]
**Describe the bug**
All the files in the extension tutorial pages are returning 4-4

**To Reproduce**
1. Go to https://developer.chrome.com/docs/extensions/mv2/getstarted/
2. Create a file called manifest.json and include the following code, or download the file **here.**
3. See 404

**Expected behavior**
Expect a file that I can download

**Comment by robdodson** [comment]
@dotproto can you take a look at this? It looks like the very first link on this page goes to a storage bucket, but the rest are broken.

**Comment by dotproto** [comment]
Yeah, this makes sense. In PR #317 I removed the references to the individual files because they were a source of errors, but I didn't make any updates to the MV2 page we cloned.
